pkg/types/build_info.go

```diff
@@ -38,6 +38,17 @@ type ErrImageBuild struct {
 	Err   error
 }
 
+func NewBuildInfo(params Params, path string, transportMethod TransportMethod, startedAt time.Time) *BuildInfo {
```

Comment:
This shouldn't accept any parameters - the user should (if they want to) set individual fields after getting the result of `NewBuildInfo`.

It should just return a buildinfo with `ExitCode` and `StartedAt` initialized.

Reply:
done